config.properties.template

```diff
@@ -77,6 +78,9 @@ sort_by_iv=false
 # Always throw curveballs
 always_curve=false
 
+# Never use berries
+never_use_berries=true
```

Comment:
I don't think so, because yellowball is harder to get than berry. if sometime I prefer using blue+berry than yellowball. But the logic you give is try yellowball first. So my conclution is
red<blue<blue+berry<yellow<yellow+berry
yours is
red<blue<yellow<yellow+berry